### PR TITLE
Leaderboard setup

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this feature we :

-Created an npm project with Webpack
-Used Gitflow
-Wrote plain HTML markup with minimum styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,187 @@
...
+
+npm install
+
+```
+
+### Usage
+
+To run the project:
+
+- Clone this repo to your local machine.
```
Comment (tugsem) [inline README.md]: - [ ] Please write the right command to start the project here as this part for usage only.

File: README.md [hunk 2]
```diff
@@ -1 +1,187 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Bleu Yves Sopoude (MrBlue)**
+
+- GitHub: [@Bleu-Yves-Sopoude](https://github.com/Bleu-Yves-Sopoude)
+
```
Comment (tugsem) [inline README.md:133]: - [ ] Please add other two accounts: twitter and linkedIn here as well to match the correct readme template.

File: README.md [hunk 3]
```diff
@@ -1 +1,187 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## Live Demo <a name="Live-demo"></a>
+
+
+- Here is the live demo link: <a href="">Live demo</a>
+
```
Comment (tugsem) [inline README.md]: - [ ] Please remove live demo as you don't have it yet. Once you have a proper link, you should add it here then.

File: README.md [hunk 4]
```diff
@@ -1 +1,187 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- **[ Drag and drop ]**
```
Comment (tugsem) [inline README.md]: - [ ] Please add two more future features here as per the requirement.

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @Bleu-Yves-Sopoude 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of gitflow
✔️bundler webpack is set up correctly
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._se._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/README.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Bleu Yves Sopoude (MrBlue)**
+
+- GitHub: [@Bleu-Yves-Sopoude](https://github.com/Bleu-Yves-Sopoude)
+- Linkedin: [@in/bleu-yves](https://www.linkedin.com/in/bleu-yves/)
+- GitHub: [@bleuYves](https://twitter.com/bleuYves)
+
```
Comment (AbelG101) [inline .github/README.md:133]: - [ ] [OPTIONAL] I believe you mistyped the Twitter handle for GitHub here as it redirects to your Twitter account. ✔

**Review by AbelG101 — APPROVED** [review]
# Hi @Bleu-Yves-Sopoude 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Webpack is configured properly ✔
- Correct use of GitHub Flow ✔
- Proper branch name✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**